### DISCUSSION TITLE
Is it possible to pass user data down in the `/register` route?

### DISCUSSION THREAD

**Discussion by Jay-fly** [fastapi-users/fastapi-users#1469] [Q&A] [answered]
Hello,

I was wondering if it’s possible to pass the user’s input data (e.g., username and password before hashing) down during the `/register` route process.

The issue I’m facing is that I need to integrate with some third-party services. As part of the user registration process, I also want to create an account for the user on these third-party services using the same username and password they provided during registration.

Currently, my approach is to override the /register route to achieve this, but I feel like this might not be the best solution. Do you have any suggestions or recommendations for a better way to implement this?

Thank you!

**Comment by hgalytoby** [comment, marked as answer]
The create method can be overridden

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L110

**Comment by alfonsodipace** [comment]
Hi @Jay-fly How did you override the register route? I'm trying to do it but without results, thanks
